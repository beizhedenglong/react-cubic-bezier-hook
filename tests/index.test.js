import { render } from 'react-testing-library'
import React from 'react'
import App from '../src/index'

test('app render', () => {
  const { container } = render(<App />)
  expect(container).toMatchSnapshot()
})
