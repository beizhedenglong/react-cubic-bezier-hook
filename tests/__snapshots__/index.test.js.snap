// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app render 1`] = `
<div>
  <div>
    hello
  </div>
</div>
`;
